python-main.js

```diff
@@ -627,13 +628,17 @@ function web_editor(config) {
             doShare();
         });
         $("#command-help").click(function () {
-            if($(".helpsupport_container").css("display") == "none"){
-                $(".helpsupport_container").css("display", "flex");
-            } else {
+            // Show help
+            $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
+            $(".helpsupport_container").css("left", $("#command-help").offset().left);
+            $(".helpsupport_container").css("display", "flex");
+        });
+        // Add document click listener
+        document.body.addEventListener('click',function(event) {
+            // Close helpsupport if the click isn't on a descendent of #command-help
+            if(!event.target.closest('#command-help'))
```

Comment:
This will exclude clicking the help button as well, which should still close the menu. We should probably give the pop-up div with class `helpsupport_lhs` an id and select that instead.